src/Uno.UI/UI/Xaml/Controls/ListViewBase/NativeListViewBase.iOS.cs

```diff
@@ -778,6 +778,56 @@ public Thickness Padding
 		}
 
 		#region Touches
+		private UIElement _touchTarget;
+
+		/// <inheritdoc />
+		public override void TouchesBegan(NSSet touches, UIEvent evt)
+		{
+			base.TouchesBegan(touches, evt);
+
+			// We wait for the first touches to get the parent so we don't have to track Loaded/UnLoaded
+			// Like native dispatch on iOS, we do "implicit captures" the target.
+			if (this.GetParent() is UIElement parent)
+			{
+				// canBubbleNatively: true => We let native bubbling occur properly as it's never swallowed by system
+				//							  but blocking it would be breaking in lot of aspects
+				//							  (e.g. it would prevent all sub-sequent events for the given pointer).
```

Comment:
```suggestion
			// We wait for the first touches to get the parent so we don't have to track Loaded/UnLoaded
			// Like native dispatch on iOS, we do "implicit captures" of the target.
			if (this.GetParent() is UIElement parent)
			{
				// canBubbleNatively: true => We let native bubbling occur properly as it's never swallowed by the system
				//							  but blocking it would be breaking in a lot of aspects
				//							  (e.g. it would prevent all subsequent events for the given pointer).
```